content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -120,10 +135,16 @@ done
 
 Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.
 
-### V2 Volume Support for ARM64 Platform
+## V2 Data Engine
+
+### Support for ARM64 Platform
 
 As of Longhorn v1.6.0, volumes using the V2 Data Engine support the ARM64 platform. For more information, see [Prerequisites](../../spdk/prerequisites/).
 
-### Upgrading Longhorn Systems with V2 Volumes
+### Upgrading Longhorn Systems
 
 Longhorn currently does not support live upgrading of V2 volumes. Ensure that all V2 volumes are detached before initiating the upgrade process.
+
+### Increase Default HugePage Size to 2 GiB
+
+To increase the quantity of v2 volumes and enhance user experience, the default HugePage Size for the V2 Data Engine has been raised to 2 GiB. Prior to the upgrade, ensure the configuration of the HugePage size to 2 GiB on each node.
```

Comment:
```suggestion
The default huge page size for the V2 Data Engine has been raised to 2 GiB, allowing the creation of more V2 volumes and enhancing the overall user experience. Before upgrading to v1.6.0, ensure that the configured huge page size on each node is 2 GiB.
```

Reply:
I used "huge page" instead of "HugePage" to align with the [Linux kernel doc](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt).